VimeoUpload/Upload/Descriptor System/Cameo/CAMUploadReqest.swift

```diff
@@ -12,15 +12,13 @@ public enum CAMUploadRequest: String
 {
     case CreateVideo
     case UploadVideo
-    case ActivateVideo
-    case VideoSettings
     case CreateThumbnail
     case UploadThumbnail
     case ActivateThumbnail
     
     static func orderedRequests() -> [CAMUploadRequest]
     {
-        return [.CreateVideo, .UploadVideo, .ActivateVideo, .VideoSettings, .CreateThumbnail, .UploadThumbnail, .ActivateThumbnail]
+        return [.CreateVideo, .UploadVideo, .CreateThumbnail, .UploadThumbnail, .ActivateThumbnail]
```

Comment:
Do you have an example of this?  I implemented the thumbnail upload based on the info that @agilefox provided in the comments on CAM-354.